packages/itwin/ec3-widget/src/components/CreateAssemblyComponent.scss

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+  .create-assembly-step {
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    .assembly-list {
+      height: calc(100vh - 248px);
+      overflow: auto;
+      .assembly-expandable-block {
+        margin-top: 8px;
+        .iui-svg-icon {
+          display: flex;
+          flex-direction: row-reverse;
+        }
+      }
+    }
+    .loading-indicator {
+      align-self: center;
+    }
+    .stepper-footer {
+      margin-top: 8px;
+      align-self: center;
+      .footer-button {
+        margin-right: 8px;
+      }
+    }
+    .button-row-above-stepper {
+      justify-content: flex-end;
+      display: flex;
+    }
+  }
```

Comment:
Formatting is off and the classes aren't prefixed with `ec3w`

Reply:
done